### PR TITLE
Use `os.userInfo()` if available

### LINKED ISSUE #13: Use `os.userInfo().username`
Why doesn't this use `os.userInfo().username` if available?

### PR DESCRIPTION
**Pull request by sonicdoe**
Closes #13.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
+import os from 'os';
 import test from 'ava';
 
 test.serial('username()', async t => {
+	os.userInfo = () => ({username: 'unicorn'});
```
Comment (sindresorhus) [inline test.js]: Can you make a separate test for this? Without the `process.env` thing. Same with the sync version too.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

**Comment by sonicdoe** [comment]
Since `os.userInfo()` takes precedence over environment variables, I can’t simply keep the test as it is. I thought of two things:

1. `delete os.userInfo` so that environment variables are used.
2. Skip those tests when running Node.js v6 or later.

Which method would you prefer? Given that I’m overwriting `os.userInfo` in the new tests, I’d opt for deleting `os.userInfo` in the existing tests. I have pushed this option if you want to take a look at it.

**Comment by sindresorhus** [comment]
Go with `1`.

**Review by sindresorhus — APPROVED** [review]

**Comment by sindresorhus** [comment]
Looks good. Thanks again :)

**Pull request merged**